From bf6dd7fa366c84544dc8a06ca68e892fdac31d60 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 28 Aug 2010 16:48:22 +0000
Subject: [PATCH 012/160] builtin.h: Include gettext.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change builtin.h to include gettext.h. This is needed in order to
translate Git's builtin commands (like git-init), which don't include
gettext.h otherwise.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/builtin.h b/builtin.h
index 0398d24..26fe6b7 100644
--- a/builtin.h
+++ b/builtin.h
@@ -6,6 +6,7 @@
 #include "cache.h"
 #include "commit.h"
 #include "notes.h"
+#include "gettext.h"
 
 extern const char git_version_string[];
 extern const char git_usage_string[];
-- 
1.7.3.272.g31195

